library.py

```diff
@@ -62,10 +67,11 @@ def check_out(filename, isbn, s_id):
 
     # Find a way to mark a book as checked out. Be sure to associate
     # the book with the student who borrowed it!
-
-
+    
+     f.__setitem__("filename",checked out) 
```

Comment:
Do you know what the magic method, __setitem__ does?

I'm pretty confident you haven't tested this.